src/model/Question.java

```diff
@@ -141,35 +141,29 @@ public void setCategoryPathName(int path) {
 
      }
 
-    /**This method randomly generates a question ID number*/
+    /**This method randomly generates a question ID number.*/
      public int selectQuestion(){
          Random random = new Random();
 
          if (categoryID == 1) { //Geek Out
-             range = (int )(Math.random() * (120 - 101) + 101);
-             System.out.println("hi1 " +range);
+             randomID = (int )(Math.random() * (120 - 101) + 101);
```

Comment:
// We'll address this in a later pull request -- will require refactoring later to capture the number of questions in a given file